src/pages/RFOX/components/Unstake/UnstakeStatus.tsx

```diff
@@ -48,28 +82,37 @@ export const UnstakeStatus: React.FC<UnstakeRouteProps> = () => {
           title: 'common.success',
           body: [
             'RFOX.unstakeSuccess',
```

Comment:
We'll want to replace "have" with "has":

<img width="323" alt="Screenshot 2024-05-27 at 11 03 13 AM" src="https://github.com/shapeshift/web/assets/97164662/f68bc0e5-52cf-43e2-9cd8-56698ca449e4">

Reply:
Fixed in https://github.com/shapeshift/web/pull/6957/commits/9188258bbe30a8a207793ba3d84dee660b1b584f, though note this does not handle plural's the event that the unstake period changes to such.

Reply:
Precisely this re: plural @0xApotheosis, the 1 hour is for dev only and the actual prod feature will not have such a low cooldown.